src/jit/hwintrinsicxarch.cpp

```diff
@@ -545,10 +545,34 @@ bool Compiler::compSupportsHWIntrinsic(InstructionSet isa)
                                                     isFullyImplmentedISAClass(isa));
 }
 
-static bool isTypeSupportedForIntrinsic(var_types type)
+//------------------------------------------------------------------------
+// hwIntrinsicSignatureTypeSupported: platform support of hardware intrinsics
+//
+// Arguments:
+//    retType - return type
+//    sig     - intrinsic signature
+//    flags   - flags of the intrinsics
+//
+// Return Value:
+//    On 32-bit platforms, return true if the intrinsic does not use any 64-bit registers (r64)
```

Comment:
This description doesn't reflect the name or the real purpose. I think it should say:
```
// Return Value:
//    Returns true iff the given type signature is supported
//
// Notes:
//    - This is only used on 32-bit systems to determine whether the signature uses no 64-bit registers.
//    - The `retType` is passed to avoid another call to the type system, as it has already been retrieved.